core-plugins/src/main/java/co/cask/hydrator/plugin/batch/aggregator/DedupConfig.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.hydrator.plugin.batch.aggregator;
+
+import co.cask.cdap.api.annotation.Description;
+import co.cask.cdap.api.data.schema.Schema;
+import co.cask.hydrator.plugin.batch.aggregator.function.First;
+import co.cask.hydrator.plugin.batch.aggregator.function.Last;
+import co.cask.hydrator.plugin.batch.aggregator.function.Max;
+import co.cask.hydrator.plugin.batch.aggregator.function.Min;
+import co.cask.hydrator.plugin.batch.aggregator.function.RecordAggregateFunction;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Splitter;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nullable;
+
+/**
+ * Plugin Configuration
+ */
+public class DedupConfig extends AggregatorConfig {
+
+  @Description("Optional comma-separated list of fields to perform the distinct on. If none is given, each record " +
+    "will be taken as is. Otherwise, only fields in this list will be considered.")
```

Comment:
This description and property name are not enough for me to understand what this plugin does.  Please update this instead of copy-pasting from distinct.